ibis/expr/types.py

```diff
@@ -192,14 +194,24 @@ def execute(self, limit='default', params=None, **kwargs):
           Pass an integer to effect a specific row limit. limit=None means "no
           limit". The default is whatever is in ibis.options.
 
+        timecontext: Tuple(pd.Timestamp, pd.Timestamp) or None, default None
+           Defines a time range of [begin, end). If table contains a time
+           column of Timestamp type, execute with data within the range of
+           timecontext. This is conceptually same as a time filter.
+           Note:
```

Comment:
I would probably remove the "Note: " here. Don't think it's necessary.

Reply:
Sound good, I removed it.